frontend/packages/topology/src/operators/TopologyServiceBindingRequestPanel.tsx

```diff
@@ -16,14 +19,31 @@ type TopologyServiceBindingRequestPanelProps = {
   edge: OdcBaseEdge;
 };
 
-const TopologyServiceBindingRequestPanel: React.FC<TopologyServiceBindingRequestPanelProps> = ({
-  edge,
-}) => {
+type PropsFromState = {
+  selectedDetailsTab?: any;
+};
+
+type PropsFromDispatch = {
+  onClickTab?: (name: string) => void;
+};
+
+const stateToProps = ({ UI }): PropsFromState => ({
+  selectedDetailsTab: UI.getIn(['overview', 'selectedDetailsTab']),
+});
+
+const dispatchToProps = (dispatch): PropsFromDispatch => ({
+  onClickTab: (name) => dispatch(UIActions.selectOverviewDetailsTab(name)),
+});
+
+const ConnectedTopologyServiceBindingRequestPanel: React.FC<PropsFromState &
+  PropsFromDispatch &
+  TopologyServiceBindingRequestPanelProps> = ({ edge, onClickTab, selectedDetailsTab }) => {
```

Comment:
Not sure I understand what is happening here... why this change? Seems like new functionality for an I18n PR.